2019/12.md

```diff
@@ -71,6 +71,8 @@ Supporting materials includes slides, a link to the proposal repository, a link
 
     | ✓ | stage | timebox | topic | presenter |
     |:-:|:-----:|:-------:|-------|-----------|
+    |   | 3     | 45m     | [Optional Chaining](https://github.com/tc39/proposal-optional-chaining/) for Stage 4 | Daniel Rosenwasser |
+    |   | 3     | 45m     | [Nullish Coalescing](https://github.com/tc39/proposal-nullish-coalescing/) for Stage 4 | Daniel Rosenwasser |
```

Comment:
Is this much time needed?

Reply:
better to overestimate :-) it can always be dropped down.